Bottom: e2f8b08cb62189831e67485295fb9de710d13384
Top:    f39d07c4ee609184f4ee534cca332161d568ca2b
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-01-23 18:35:34 +0000

Hack around Circuit CAM / EDG brokeness reading drill files






---

diff --git a/src/hid/gerber/gerber.c b/src/hid/gerber/gerber.c
index d7290be..814b3c1 100644
--- a/src/hid/gerber/gerber.c
+++ b/src/hid/gerber/gerber.c
@@ -462,10 +462,17 @@ gerber_set_layer (const char *name, int group)
 	      int ap = findApertureCode (pending_drills[i].diam, ROUND);
 	      fprintf (f, "T%02d\015\012", ap);
 	    }
+#ifdef EDG_ARE_BROKEN
+	  fprintf (f, "X%06ldY%06ld\015\012",
+		   100 * gerberX (PCB, pending_drills[i].x), 100 * gerberY (PCB,
+								pending_drills
+								[i].y));
+#else
 	  fprintf (f, "X%06ldY%06ld\015\012",
 		   gerberX (PCB, pending_drills[i].x), gerberY (PCB,
 								pending_drills
 								[i].y));
+#endif
 	}
       free (pending_drills);
       n_pending_drills = max_pending_drills = 0;
